### PR TITLE
push submissions to ASP

### PR DESCRIPTION
**Pull request by jamesdotcom**

### CODE REVIEW

File: src/java/main/com/topcoder/direct/services/view/action/project/ProjectWorkManagementAction.java [hunk 1]
```diff
@@ -0,0 +1,946 @@
...
+
+        for (ContestCopilotDTO copilot : copilots) {
+            if (copilot.getUserId() == currentUser.getUserId()) {
+                // pass permission checking
+                return;
+            }
+        }
+
+        // still not pass permission checking, throw exception
+        throw new Exception("You don't have permission to access");
```
Comment (TriangleIL) [inline src/java/main/com/topcoder/direct/services/view/action/project/ProjectWorkManagementAction.java]: Do we need to add the work owner here as well?

File: src/java/main/com/topcoder/direct/services/view/action/project/ProjectWorkManagementAction.java [hunk 2]
```diff
@@ -0,0 +1,946 @@
...
+        try {
+
+            String tcjwt = getJWTCookieValue();
+
+            checkPermission();
+
+            ASPClient aspClient = ASPClientFactory.build(tcjwt);
+
+            // get the worksteps via API
+            // this.workSteps = aspClient.retrieveWorkSteps(String.valueOf(getFormData().getProjectId()));
```
Comment (TriangleIL) [inline src/java/main/com/topcoder/direct/services/view/action/project/ProjectWorkManagementAction.java]: Does this work or is it hard coded for now?

Comment (jamesdotcom) [inline src/java/main/com/topcoder/direct/services/view/action/project/ProjectWorkManagementAction.java]: it should work. i'll uncomment

File: src/java/main/com/topcoder/direct/services/view/action/project/ProjectWorkManagementAction.java (hunk 1 above)
Comment (jamesdotcom) [inline src/java/main/com/topcoder/direct/services/view/action/project/ProjectWorkManagementAction.java]: That would be the customer right? I don't think they should be able to access this page.

**Pull request merged**